### DISCUSSION TITLE
profe, tengo una duda y un problema

### DISCUSSION THREAD

**Discussion by sophienunez** [sebasvelasco353/FDP-2022_01#45] [Q&A]
El problema que tengo es que aunque usé el mismo código que nos estabas mostrando en clase, cambié el submit a button y además revisé 10 veces la ortografía, la consola no me está leyendo los datos y pues con esto no he podido revisar el condicional de +50.

La duda es que estaba haciendo igual lo del condicional con el disabled y luego poner el disabled=false con el if pero no sé cómo meter un imput diferente en el condicional del imput de la edad.

Lo ando haciendo así (no le he puesto el else) pero igual no he podido revisar si sirve o no por lo de la consola:

var edadSelect = document.getElementById("edad");
for (let i = 0; i < edadSelect.length; i++) {
    const element = edadSelect[i];
    if (edad => 50) {
        let htmlElement = document.getElementById("pruebasc").disabled = false;
    }
}

Se me ocurrió llamar el otro imput así pero no sé porqué siento que no es la manera correcta o que le falta algo.
ayuda xfa :')

**Comment by sebasvelasco353** [comment]
Hello, ese for no lo necesitas porque no vas a recorrer un arreglo y como addSelect no es un arreglo te da un error, entonces al borrar el for y el elemen = ... vas a poder entrar en el if.

**Reply by sophienunez** [reply]
oohh, vale profe, lo voy a intentar. de resto si me sirve llamar el input así? y me puede servir mientras tanto a pesar de la consola para ver si me va quedando bien o no?

**Reply by sophienunez** [reply]
no m sirve DD':

**Reply by sophienunez** [reply]
lo dejé así: 

var edad = document.getElementById("edad");
(let i = 0; i < edad.length; i++) {
    if (edad => 50) {
        let htmlElement = document.getElementById("pruebasc").disabled = false;
    }
}

**Reply by sophienunez** [reply]
pero no m sirve y la consola no m coge nada ayuda profe DDD':

**Reply by sophienunez** [reply]
la consola me dice Uncaught SyntaxError: Unexpected identifier

**Reply by sebasvelasco353** [reply]
si, siempre y cuando el elemento tenga ese mismo id en el html debe funcionarte sin problemas, pero para quitar el disabled no usas .disabled porque no es una funcion de ese elemento sino que usas .removeAttribute("disabled");

Para que te imprima en la consola usas console.log(<nombre de variable o valor que quieres usar>); y lo pones donde quieres que imprima algo en la consola.

**Reply by sebasvelasco353** [reply]
(let i = 0; i < edad.length; i++) { <--- esto tambien hace parte del for.
una vez borres eso y la llave que cierra el for al final deberia funcionarte.

**Reply by sophienunez** [reply]
ya, ya lo del disabled fue que busqué en internet cómo hacerlo y me decía que así xdn't
gracias por contestar profe :')

**Reply by sebasvelasco353** [reply]
Me alegro que te haya servido!

OMG, acabo de hacerlo en una pagina y si funciona, no sabia, GRACIAS!


<img width="802" alt="Screen Shot 2022-02-15 at 10 54 50 PM" src="https://user-images.githubusercontent.com/11669998/154193322-bab62847-7500-46c6-9bec-568900194212.png">

**Reply by sophienunez** [reply]
a mí todavía no me sirve profe auxilio :'(

**Reply by sophienunez** [reply]
var edad = document.getElementById("edad");
    if (edad => 50) {
        let htmlElement = document.getElementById("pruebasc").removeAttribute("disabled");
    } 
    else if (edad < 50) {
        let htmlElement = document.getElementById("pruebasa").removeAttribute("disabled");
    }

lo tengo así y ahora se me quitó el disabled del primero permamentemente, no dependiendo del número. el mayor o menor sí se usa así?

**Comment by sebasvelasco353** [comment]
el mayor o igual lo escribes asi >= y se me habia pasado que no necesitas declarar la variable htmlElement, en ese caso solo vas a hacer el disabled.

**Reply by sophienunez** [reply]
ya, o sea borro let htmlElement = ?

**Reply by sebasvelasco353** [reply]
ajam, borras solamente "let htmlElement ="

**Reply by sophienunez** [reply]
no profe, nada. lo dejé así pero na

var edad = document.getElementById("edad");
    if (edad >= 50) {
        document.getElementById("pruebasc").disabled = false;
    } 
    else if (edad < 50) {
        document.getElementById("pruebasa").disabled = false;
    }

ya no sé ni qué tengo mal

**Reply by sebasvelasco353** [reply]
recuerda que tienes que obtener el valor del input edad, no solo el elemento.

**Reply by sophienunez** [reply]
no sé qué siginifica eso pero ya googleo, perdona tanta jodedera, ya voy a tratar de sacar eso de una vez

**Reply by sebasvelasco353** [reply]
Hahaha tranquila, no te preocupes que yo les dije que iba a estar hoy hasta esta hora en el compu.

Exitos!

**Comment by sophienunez** [comment]
profe, se me olvidó avisarte ayer que subí el trabajo a el mismo repositorio en el que está lo de la página web de la vez pasada, git no me quiere funcionar, todo sale rojo y error

**Reply by sebasvelasco353** [reply]
Puedes porfa mostrarme que errores te están saliendo?
